Microsoft.Toolkit.Uwp.UI.Controls/TabbedCommandBar/TabbedCommandBarItem.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// A <see cref="CommandBar"/> to be displayed in a <see cref="TabbedCommandBar"/>
+    /// </summary>
+    [TemplatePart(Name = "PrimaryItemsControl", Type = typeof(ItemsControl))]
+    [TemplatePart(Name = "MoreButton", Type = typeof(Button))]
+    public class TabbedCommandBarItem : CommandBar
+    {
+        private ItemsControl _primaryItemsControl;
+        private Button _moreButton;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TabbedCommandBarItem"/> class.
+        /// </summary>
+        public TabbedCommandBarItem()
+        {
+            DefaultStyleKey = typeof(TabbedCommandBarItem);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Header"/> property.
+        /// </summary>
+        public static readonly DependencyProperty HeaderProperty = DependencyProperty.Register(
+            nameof(Header),
+            typeof(string),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(string.Empty));
+
+        /// <summary>
+        /// Gets or sets the title of this ribbon tab.
+        /// </summary>
+        public string Header
+        {
+            get => (string)GetValue(HeaderProperty);
+            set => SetValue(HeaderProperty, value);
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="Footer"/> property.
+        /// </summary>
+        public static readonly DependencyProperty FooterProperty = DependencyProperty.Register(
+            nameof(Footer),
+            typeof(UIElement),
+            typeof(TabbedCommandBarItem),
+            new PropertyMetadata(new Border()));
```

Comment:
Reason for defaulting to a Border here? I was trying to see where this was referenced in the XAML?

Reply:
I just didn't want it to default to `null`, is that what it should do instead?

Reply:
I think as long as there's not visual glitches or anything, the Footer being null by default is fine. [As an example the default value for ListViewBase.Footer is null](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.listviewbase.footer). [NavigationView.PaneFooter is also null](https://docs.microsoft.com/uwp/api/windows.ui.xaml.controls.navigationview.PaneFooter).

Reply:
Setting it to null didn't cause any issues. However, I removed the Footer property for now since it's not implemented. Adding it would require a retemplate of the CommandBar, which I really want to avoid at least until https://github.com/microsoft/microsoft-ui-xaml-specs/pull/108 is finalized.